### PR TITLE
[source-stripe] 🐛 bump cdk to 5.10.2 to stop using PrintBuffer due to record count mismatches

### PR DESCRIPTION
**Pull request by brianjlai**
## What

The CDK was updated in `5.10.2` to not have `entrypoint.py` use the PrintBuffer. I tested this against a pre-release version which appear to get rid of the mismatches. This is just the bump of `source-stripe` to update the change

## User Impact

n/a

## Can this PR be safely reverted and rolled back?

- [x] YES 💚
- [ ] NO ❌

### CODE REVIEW

**Review by artem1205 — APPROVED** [review]

**Review by bazarnov — APPROVED** [review]

**Review by maxi297 — APPROVED** [review]
:shipit:

File: airbyte-integrations/connectors/source-stripe/integration_tests/expected_records.jsonl [hunk 1]
```diff
@@ -49,7 +49,6 @@
...
...dCl3kj","tax_rates":[]}],"has_more":false,"total_count":1.0,"url":"/v1/subscription_items?subscription=sub_1POL30EcXtiJtvvh4UdCl3kj"},"latest_invoice":"in_1PYUP7EcXtiJtvvhSPc5z2FG","livemode":false,"metadata":{},"next_pending_invoice_item_invoice":null,"on_behalf_of":null,"pause_collection":null,"payment_settings":{"payment_method_options":null,"payment_method_types":null,"save_default_payment_method":"off"},"pending_invoice_item_interval":null,"pending_setup_intent":null,"pending_update":null,"plan":{"id":"price_1PDtzkEcXtiJtvvhY4aqjMSq","object":"plan","active":true,"aggregate_usage":null,"amount":null,"amount_decimal":null,"billing_scheme":"tiered","created":1715111292,"currency":"usd","interval":"week","interval_count":1,"livemode":false,"metadata":{},"meter":null,"nickname":null,"product":"prod_Q423JP7oX7TF5U","tiers_mode":"graduated","transform_usage":null,"trial_period_days":null,"usage_type":"licensed"},"quantity":1,"schedule":null,"start_date":1717598562,"status":"past_due","test_clock":null,"transfer_data":null,"trial_end":null,"trial_settings":{"end_behavior":{"missing_payment_method":"create_invoice"}},"trial_start":null,"updated":1717598562},"emitted_at":1720515564910}
```
Comment (brianjlai) [inline airbyte-integrations/connectors/source-stripe/integration_tests/expected_records.jsonl:52]: i have no idea why this record got removed from our stripe account, but I couldn't retrieve it listing subscriptions via http requests and trying the direct request using the `id` using `https://api.stripe.com/v1/subscriptions/sub_1MowQVLkdIwHu7ixeRlqHVzs`, resulted in a 404, so I don't believe it exists anymore.

This was passing last night/this morning so i'm convinced this isn't an issue with my changes

**Pull request merged**